javascript/fallback.js

```diff
@@ -0,0 +1,68 @@
+function fallbackWarning() {
+  console.warn(
```

Comment:
Is there a way to run this just once? Printing to console will be a lot more expensive, than doing the computation, even in JS.

Reply:
Maybe when we detect the fallback is needed? Although I do not know how to detect when it is needed...